meta-balena-common/recipes-containers/balena/balena_git.bb

```
@@ -54,6 +54,7 @@ FILES_${PN} += " \
 	/home/root \
 	/boot/storage-driver \
 	${localstatedir} \
+	${libexecdir} \
```

```suggestion
	${bindir} \
```
This should match the install path in case they don't resolve to the same thing.

---

Right, that must have been a left over.